components/script/dom/bindings/typedarrays.rs

```diff
@@ -31,6 +31,32 @@ unsafe impl<T> crate::dom::bindings::trace::JSTraceable for HeapTypedArray<T> {
     }
 }
 
+pub fn new_initialized_heap_typed_array<T>(init: HeapTypedArrayInit) -> HeapTypedArray<T>
```

Comment:
Let's return a result, and return the error from `create_typed_array_with_length`. (creating the buffer is falling, as we saw in the other PR).

Reply:
done